### PR TITLE
Ignore -named_curve auto value to improve backwards compatibility

### LINKED ISSUE #3490: auto option for -named_curve switch missing in man pages
Seems that 'auto' option is missing in man pages under -named_curves, for current master and 1.1.0 also.

### PR DESCRIPTION
**Pull request by t8m**
Fixes #3490

### CODE REVIEW

File: ssl/ssl_conf.c [hunk 1]
```diff
@@ -227,6 +227,12 @@ static int cmd_ECDHParameters(SSL_CONF_CTX *cctx, const char *value)
     EC_KEY *ecdh;
     int nid;
 
+    /* Ignore values supported by 1.0.2 for the automatic selection */
+    if ((cctx->flags & SSL_CONF_FLAG_FILE) && !strcasecmp(value, "+automatic"))
+        return 1;
+    else if ((cctx->flags & SSL_CONF_FLAG_CMDLINE) && !strcmp(value, "auto"))
```
Comment (richsalz) [inline ssl/ssl_conf.c]: no else after return :)

Comment (richsalz) [inline ssl/ssl_conf.c]: oops, sorry, and explicitly test against zero, don't use !

**Comment by t8m** [comment]
Updated patch

**Comment by mattcaswell** [comment]
Ping @richsalz

**Comment by richsalz** [comment]
fnx for the nudge.

**Comment by mattcaswell** [comment]
Pushed. Thanks.

**Pull request closed without merging**